numpy/core/defchararray.py

```diff
@@ -1187,7 +1187,15 @@ def replace(a, old, new, count=None):
     See Also
     --------
     str.replace
+    
+    Examples
+    --------
+    >>> import numpy as np 
+    >>> print(np.char.replace('That is a mango', 'mango', 'banana'))
+    That is a banana
 
+    >>> print(np.char.replace('The dish is fresh', 'is', 'was'))
+    The dwash is fresh
```

Comment:
```suggestion
    The dwash was fresh
```

One approach is to suggest a change on your own pull request like above, from "Files Changed" tab, and then commit it through the interface.